packages/frontend/src/providers/ExplorerProvider.tsx

```diff
@@ -1125,6 +1194,12 @@ function reducer(
                 );
             return {
                 ...state,
+                metadata: {
+                    tableCalculations:
+                        state.metadata?.tableCalculations?.filter(
+                            (tc) => tc.name !== action.payload,
+                        ),
+                },
```

Comment:
nice - is there somewhere we can _clear_ this `metadata` property to keep it clean? 
Like on `discard` and/or on `save` 🤔

Reply:
I believe setting it on `update` and removing on `tc remove` should be enough as it will always be the source of truth after every TC update on the explore.
I tried clearing it after the request to get the results but the problem is that after the 3rd table calculation update it will go back to a default value since it finds another mismatch in the name and doesn't have the metadata to update it in the hook :(

Reply:
I just seen that you've added the filtering on removal of a TC - thanks!